Apromore-Custom-Plugins/Log-Animation-Portal-Plugin/src/main/java/org/apromore/plugin/portal/loganimation/LogAnimationPlugin.java

```diff
@@ -79,6 +79,11 @@ public void setGroupLabel(String groupLabel) {
         this.groupLabel = groupLabel;
     }
 
+    @Override
+    public String getIconPath() {
+        return "loganimation/icon.svg";
+    }
+
```

Comment:
The same comment for the same PR in EE repo: this behavior should be made default, only plugin with exception in icon or icon path should override this behavior. This would reduce much boilerplate code and missing errors.